src/domains/agents/decision/decision_agent.py

```
@@ -285,6 +297,307 @@ async def learn(self, beliefs: List[Belief], intentions: List[Intention], contex
+                        decision = self._create_decision_from_dict(current_decision, opportunities)
+                        if decision:
+                            decisions.append(decision)
+                    current_decision = {'description': line}
+                
+                elif 'decision_type:' in line.lower() or 'decision:' in line.lower():
+                    decision_type = line.split(':', 1)[1].strip().lower()
+                    current_decision['decision_type'] = decision_type
+                elif 'priority:' in line.lower():
+                    try:
+                        current_decision['priority'] = int(line.split(':', 1)[1].strip())
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)